app/src/settings/development.py

```diff
@@ -163,3 +164,30 @@
 
 # Especifica os esquemas permitidos no ambiente de desenvolvimento
 REST_FRAMEWORK_SCHEMAS = ["http"]
+
+
+# Configurações de Logging
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'handlers': {
+        'console': {
+            'level': 'WARNING',
+            'class': 'logging.StreamHandler',
+            'stream': os.sys.stdout,
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (code_refinement):** Consider configuring log file output for better log management.

Directing logs to stdout is convenient for development but consider also configuring a file handler for more persistent log management, aiding in debugging past issues.

```suggestion
            'stream': os.sys.stdout,
            'handlers': ['console', 'fileHandler'],
        },
    },
    'handlers': {
        'fileHandler': {
            'level': 'WARNING',
            'class': 'logging.FileHandler',
            'filename': 'app_logs.log',
        },
```